### ISSUE TITLE
Make Database Navigator Auto-refresh Possible (and default)

### ISSUE THREAD

**Issue by NoahZielke** [dbeaver/dbeaver#20845]
**Is your feature request related to a problem? Please describe.**
I'm brand new to dbeaver. I created a table (in Postgres), and couldn't figure out why it wasn't appearing in the Database Navigator pane on the left. Turns out I had to manually right click, and hit "refresh". I think this is clunky and unintuitive. Why wouldn't this be automatic?

**Describe the solution you'd like**
An option to enable auto-refresh in Database Navigator, and/or make auto-refresh the default

**Describe alternatives you've considered**
Manually refreshing whenever I make a table

**Additional context**
n/a

**Comment by E1izabeth** [comment]
Thank you for the issue. Could you clarify what do you mean by auto-refresh? Are you talking about functionality of reloading metadata on DDL execution or refreshing navigator by timer?

**Comment by NoahZielke** [comment]
Hello,
I'm not 100% sure which of those two fits what I'm describing. I'm just referring to having this table name appear when I create a table in a script:

![image](https://github.com/dbeaver/dbeaver/assets/22875039/2fbe08a5-73d5-4023-aa21-e8df9d6a0927)

**Comment by E1izabeth** [comment]
I see.
I'll close this issue as a duplicate of #7924. You can vote for the issue with 👍🏻 under the issue description.

**Comment by NoahZielke** [comment]
Ah. So that's effectively a wontfix. That issue is 3 years old, I couldn't find it when searching for how to resolve this, and it doesn't seem like anyone else cares. That's too bad.

**Comment by E1izabeth** [comment]
We actually were working on it recently, but have found some blockers that should be done before releasing that feature. We keep it in mind and plan to finish in future.
